test/sqatt/src/Sqatt.hs

```diff
@@ -191,17 +190,17 @@ toFSSafeStr str = repl <$> str
 -- Throws an exception on failure.
 checkSMTSolvers :: IO ()
 checkSMTSolvers = do
-  putStrLn "WARNING: The presence of SMT solvers was not checked."
-  putStrLn "         First issue #47 needs to be resolved."
-  putStrLn "See: https://github.com/TorXakis/TorXakis/issues/47"
-
+  traverse_ checkCommand txsSupportedSolvers
+  return ()
+  where
+    txsSupportedSolvers = Prelude.map addExeSuffix ["z3","cvc4"]
```

Comment:
Just for the record: I agree with these changes, as long as this is a temporary fix. The executable names should be extracted from the `TorXakis` configuration ...

Reply:
I agree. I thought it'd be better to do that as part of #362